### ISSUE TITLE
Improve fps Resnet model

### ISSUE THREAD

**Issue by abdou31** [tensorflow/tensorflow#31099]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10 x64
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below):1.14.1
- Python version:3.6.8
- CUDA/cuDNN version:9.0/7.4
- GPU model and memory: Nvidia GeForce 840m

I have trained a Resnet Model from scratch to detect eye region in real-time.
The trained model gives me some results but the values fps <10.
I have opened the webcam and load Tensorflow model using a python script and OpenCV library.
I tried many solutions ( decrease the width and height of the input image, set the batch size to 1 ...) but any of those methods working.
I want to know if there is a solution to increase the value of fps?

**Comment by lufol** [comment]
Hi abdou31,
you could try the tensorflow slim models. 
[Here](https://github.com/tensorflow/models/tree/master/research/slim) is an overview how to use them. There is even a pretrained model that might be a good start to try out the performance before training your own network.

**Comment by abdou31** [comment]
I just want to tell you that I have used a Github Repository ( project ) that use a CNN to create a Resnet model to detect 68 facial landmarks.
NB: The output of the CNN is Regression, not a classification.
So I have used this CNN to train my own dataset to detect 40 eye region landmarks, I tried to use the pre-trained model that he gives but it doesn't work because of the number of landmarks ( units of logits layers ) and the width and height of the input image tensor are differents.
For that reason, I should use Transfer learning ( that I didn't how can I start using this technique ).
The models that you give are many and are different, I would like to know:
+ How can I choose a pre-trained model that can be compatible with my CNN?
+ How can I use transfer learning to adjust some parameters in the CNN?
Thanks

**Comment by lufol** [comment]
Since the project of the github repository you linked seems to be specific, I would consider converting the network you have trained to a tf lite network and try the performance then.

**Comment by abdou31** [comment]
I have converted to tflite and I did not how can I use tflite model for prediction

**Comment by lufol** [comment]
Sorry, I mixed something up there. Tf lite is just for embedded devices, but I guess you are trying to run the model on your pc. 
You can have a look at https://www.tensorflow.org/guide/performance/overview 
If this does not help you, you might consider opening a question on stackoverflow since this is not an issue with tf.

**Comment by abdou31** [comment]
The link that you post doesn't meet for what I need and what I want.
The performance should be good on the phone because the SDK that I want should be performant and the value of fps should be heigher.
I know that this is related to the GPU or CPU of the phone you test on it.
But there many application that use machine learning SDK work perfectly and fast like BinaryFace SDK ... 
I know just gpu delegate on.the phone that can improve the performance but I don't know how to use it.
What can improve the performance ?

**Comment by ymodak** [comment]
See step by step tutorial for GPU delegate on android https://www.youtube.com/watch?reload=9&v=Xkhgre8r5G0&feature=youtu.be
